README.md

```diff
@@ -92,6 +92,22 @@ If you use MySQL 8, please read [this link](https://www.php.net/manual/en/mysqli
 
 7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.
 
+### Upgrade Step
+
+It is recommended to do a backup of the Database before upgrading the database schemas.
+If you have installed a previous version Loris, you can upgrade Loris to the latest in the following ways:
+
+1. Using Git command checkout to the latest version of Loris. 
+ ```bash
+ git checkout aces/22.0-release
```

Comment:
this is on the master branch.  this number in this readme will need to be updated right before every release.  Can we find an easier way to write/maintain this?

Reply:
There's also no guarantee the remote will be named `aces`.